composer.json

```
@@ -26,12 +26,13 @@
         }
     },
     "require": {
-        "php": "^7.0",
+        "php": "^7.0 || ~8.1",
         "symfony/config": "^2.8 || ^3.0 || ^4.0",
         "symfony/console": "^2.8 || ^3.0 || ^4.0",
+        "symfony/http-client": "^5.4",
         "symfony/http-kernel": "^2.8 || ^3.0 || ^4.0",
         "symfony/dependency-injection": "^2.8,>=2.8.51 || ^3.0,>=3.4.26 || ^4.1,>=4.1.12 || ^4.2,>=4.2.7",
-        "symfony/process": "^2.8 || ^3.0 || ^4.0",
+        "symfony/process": "^2.8 || ^3.0 || ^4.0 || ^5.4",
```

```suggestion
        "symfony/process": "^2.8 || ^3.0 || ^4.0 || ^5.0",
```
No features strictly from 5.4 are used, so the expectation can be lowered to 5.0